support/managing-cluster-resources.adoc

```diff
@@ -0,0 +1,13 @@
+[id='managing-cluster-resources']
+= Managing your cluster resources
+include::modules/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::modules/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: managing-cluster-resources
+
+toc::[]
+
+You can apply global configuration in {product-title}. Operators apply this configuration across the cluster.
```

Comment:
The first sentence makes it sound like "global configuration" is a feature, so I'd try clarifying a bit. Something like this for example:
```suggestion
You can apply global configuration options in {product-title}. Operators apply these configuration settings across the cluster.
```

Reply:
acked